tap_google_ads/streams.py

```diff
@@ -7,7 +7,7 @@
 from singer import utils, metrics
 from google.protobuf.json_format import MessageToJson
 from google.ads.googleads.errors import GoogleAdsException
-from google.api_core.exceptions import ServerError, TooManyRequests
+from google.api_core.exceptions import ServerError, TooManyRequests, Unauthorized
```

Comment:
looking at the error in [CCI logs](https://app.circleci.com/pipelines/github/singer-io/tap-google-ads/1021/workflows/17da1986-3306-4f05-b15c-84fbbc095dba/jobs/4127?invite=true#step-103-2745), the import has to be `Unauthenticated`